include/tbbexec/tbb_thread_pool.hpp

```diff
@@ -0,0 +1,416 @@
+#pragma once
+
```

Comment:
This file needs a copy of the license, as other files in the repo. Since you copied the `static_thread_pool` code and hacked it, you should have two copyright lines: one for NVIDIA and one for yourself.